src/components/TopologySelectorModal/TopologySelectorModal.js

```diff
@@ -0,0 +1,92 @@
+import React, { useState } from "react";
+import { v4 as uuid } from "uuid";
+import PropTypes from "prop-types";
+import { TopologyCard } from "../Topology";
+import { Modal } from "../Modal";
+
+export const TopologySelectorModal = ({
+  handleModalClose,
+  isOpen,
+  footerStyle,
+  footerText,
+  topologies,
+  title,
+  titleStyle,
+  footerTextStyle,
+  buttonStyle,
+  buttonTitle,
+  setSendData
+}) => {
+  const [checked, setChecked] = useState({
```

Comment:
На этом этапе уже нужно работать с реальными данными.

defaultChecked - массив с id карточек

```suggestion
  const [checked, setChecked] = useState(defaultChecked);
```